### ISSUE TITLE
Need some information

### ISSUE THREAD

**Issue by niravpatel2008** [Rayhane-mamah/Tacotron-2#5]
Hello Rayhane,

i would like to try your program before that i would like to know few things:

- Do you have any samples generated using your program ?
- How many training steps required to get noticeable output ?
- And after how many steps it gets stable ?

**Comment by Rayhane-mamah** [comment]
Hello @niravpatel2008, thanks for reaching out and sorry for the late reply.

As you can see, this was a wild week end trying to improve the repository. to answer your questions:

- I don't have any "good" samples generated yet, but the code to generate them is already uploaded. The reason for this is that I had some problems with the attention until recently (7e67d8b43cf9c81c0abd1926c2288c9d68ab2d4e). My personal model is currently training and should be ready for synthesis soon. (please keep in mind that the vocoder has not yet been uploaded)
- In order to get noticeable output, you need around 8k~10k steps (in order to get final waveforms, i'm currently using [this repository](https://github.com/r9y9/wavenet_vocoder) as a Wavenet vocoder conditioned on my model's Mel-Spectrograms. (I don't think I'll upload any finished samples before uploading my own vocoder)
- Finally, i'm supposing you want to know after how many steps the model converges. Actually, learning curves (that I will upload [here](https://github.com/Rayhane-mamah/Tacotron-2/issues/4) shortly) show that the model actually converges extremely fast (as the MSE loss tend to 0 in ~1k steps, same for <stop_token> loss). The only loss that keeps the model improving after that is the regularization. (You'll see it better with the tensorboard curves)

So, to wrap it up, if you're looking for an entire end-to-end text to speech, you will have to wait a little bit for my vocoder or you can try the wavenet vocoder i linked to above. Otherwise, i can guarantee that by the end of this repository, you will have the exact same model as Tacotron-2 described in the paper.

Thank you for your interest in my work. let me know if i can be of any further assistance.

**Comment by Rayhane-mamah** [comment]
I am closing this issue as this matter is now discussed [here](https://github.com/Rayhane-mamah/Tacotron-2/issues/4).
